.travis.yml

```diff
@@ -14,7 +14,7 @@ before_script:
   - . vendor/ci/travis/static-analysis.sh
 
 script:
-  - phpunit $PHPUNIT_FLAGS
+  - php vendor/bin/phpunit $PHPUNIT_FLAGS
```

Comment:
In my opinion it's better to use the one installed locally so that correct version of PHPUnit will be used.